acceptance/tests/sameness/sameness_test.go

```diff
@@ -776,3 +808,39 @@ func getPeeringAcceptorSecret(t *testing.T, cfg *config.TestConfig, server *clus
 
 	return acceptorSecretName
 }
+
+// localityForRegion returns the full api.Locality to use in tests for a given region string.
+func localityForRegion(r string) api.Locality {
+	return api.Locality{
+		Region: r,
+		Zone:   r + "a",
+	}
+}
+
+// checkLocalities checks the given cluster for client and server service instances matching the locality expected
```

Comment:
I was just thinking about this, but to avoid confusion we should probably refer to all the non server/clients as `static-server` and `static-client`. Everyone in the Consul testing world will understand immediately what's being talked about.

Reply:
Might make sense to change it in other places too, wherever it makes sense to use `static-server` and `static-client`

Reply:
Good call! Agree that would improve clarity - will do